src/components/vipps-login-button/index.tsx

```diff
@@ -0,0 +1,54 @@
+import {LoginTexts, useTranslation} from '@atb/translations';
+import React from 'react';
+import ThemeText from '@atb/components/text';
+import ThemeIcon from '@atb/components/theme-icon';
+import {TouchableOpacity, View} from 'react-native';
+import {StyleSheet, Theme} from '@atb/theme';
+import vippsLogo from '@atb/assets/svg/color/icons/ticketing/VippsLogo';
+
+export const VippsLoginButton = ({
+  onPress,
+  disabled,
+}: {
+  onPress: () => {};
+  disabled: boolean;
+}) => {
+  const {t} = useTranslation();
+  const style = useStyle();
+  const getVippsLabel = () => (
+    <View style={[style.container, disabled && style.disabledOpacity]}>
+      <ThemeText type="body__primary--bold" style={style.label}>
+        {t(LoginTexts.logInOptions.options.vipps.label)}
+      </ThemeText>
+      <ThemeIcon svg={vippsLogo} style={style.icon} />
+    </View>
+  );
+
+  return (
+    <TouchableOpacity
+      accessibilityLabel={t(LoginTexts.logInOptions.options.vipps.a11yLabel)}
+      onPress={onPress}
+    >
+      {getVippsLabel()}
+    </TouchableOpacity>
+  );
+};
+
+const useStyle = StyleSheet.createThemeHook((theme: Theme) => ({
+  container: {
+    flexDirection: 'row',
+    backgroundColor: '#FF5B24',
```

Comment:
Should this be included fromt he theme or as a named color or something? Ah, it's probably the "vipps" color? Can be nice to have it as a constant variable at the top or something